internal/eth/compiler.go

```diff
@@ -53,22 +52,22 @@ func getSolcExecutable(requestedVersion string) (string, error) {
 		solcVerChecker, _ = regexp.Compile("^([0-9]+)\\.?([0-9]+)")
 	}
 	if defaultSolc == "" {
-		if envVar := os.Getenv("KLD_SOLC_DEFAULT"); envVar != "" {
+		if envVar := os.Getenv("FLY_SOLC_DEFAULT"); envVar != "" {
 			defaultSolc = envVar
 		} else {
 			defaultSolc = "solc"
 		}
 	}
 	solc := defaultSolc
 	if v := solcVerChecker.FindStringSubmatch(requestedVersion); v != nil {
-		envVarName := "KLD_SOLC_" + v[1] + "_" + v[2]
+		envVarName := "FLY_SOLC_" + v[1] + "_" + v[2]
```

Comment:
@nguyer - as you have a chance to do the formal review, I think areas where externals changes peek through like the env vars are a key area we need to think about the potential for regressions in running environment.
Should be simple for just the external places (once collated) to have a little helper to support the old ones, without them getting in the way of the code readability.

Reply:
@jimthematrix - fyi, as you're picking up the review here 🙏 
... if we don't find many others like this, it might be possible to take the migration hit without supporting both env vars.
We already have build changes in master that will affect source level users (Go 1.16, and the split out to kldbind). So we are going to need a decent readme update (needed for a bunch of reasons), and release notes anyway. The biggest current consumption is via the Kaleido managed service, and I know we can manage this change there.